src/main/telemetry/jetiexbus.c

```diff
@@ -263,6 +326,50 @@ int32_t getSensorValue(uint8_t sensor)
         return getEstimatedVario();
         break;
 
+    case EX_GPS_SATS:
+        return gpsSol.numSat;
```

Comment:
Has it been tested that all of these values will be interpreted correctly on the TX side in the case where there is no GPS receiver present?

Reply:
If a sensor is not present or the feature is not turned on, the telemetry value will not be transmitted.

Reply:
The presence of a GPS sensor cannot be reliably detected during initialisation, since the sensor usually takes a while to initialise and start sending data, so we can only check for the feature during init. If the feature is enabled but no sensor is found, (bogus) telemetry values are sent in the current implementation.

Reply:
I assumed that the user only turns on the feature if there really is a GPS receiver.

After the initialization, the EX BUS protocol expects the sensor description before telemetry values ​​are sent.
In the initialization phase, I ask which sensors are present or whether the GPS feature is activated in order to determine the available telemetry values. Depending on that, I send the sensor descriptions to the transmitter.

If no GPS fix is ​​available, all values ​​will be sent with 0.

The variable gpsSol.numSat has only a value if a GPS fix is ​​available.

Is there perhaps another solution?

Reply:
GPS is a hard case, and there is no strictly right or wrong answer to this. You are sending the GPS sensor descriptions speculatively, based on the feature setting, as at that time you can not know if a GPS is present. For what happens from there on there are two ways to go on about it:
- send the GPS values all of the time, independent of a GPS being present or not. This will cause bogus values to be sent if there is no GPS found; (this is what you are doing now)
- check `sensor(SENSOR_GPS)` and skip sending GPS values if no GPS is found. This will require additional complexity.
I am no expert in the JetiEX protocol, so I cannot tell which of these options is the right one, I just wanted to make sure you are aware they exist.

Reply:
Ok, how about this possibility:
I send the telemetry description based on the feature selection, but disable the telemetry transmission until the sensor is found.

The transmitter displays inactive telemetry values ​​by flashing. This would also be a way to detect a non-existent or inactive GPS (in the assumption of the user, the GPS has activated only if one is present)

Reply:
Yep, that sounds good. And as you say, this will make it easy for the user to distinguish between 'no GPS found' and 'no GPS fix'.